attack_json_to_bullets.py

```diff
@@ -31,15 +31,14 @@ def get_data_from_branch(domain, branch="master"):
     args = parser.parse_args()
     # Load custom layer JSON
     # First, try for a local file
-    if path.exists(args.jsonfile) and path.isfile(args.jsonfile):
+    try:
         with open(args.jsonfile) as f:
             custom_layer = json.load(f)
-    # If not a local file, try URL
-    else:
+    except (FileNotFoundError, IsADirectoryError):
```

Comment:
I think this needs to be, `except (FileNotFoundError, IsADirectoryError, OSError):` to work when querying remote URL (the URL is invalid argument).

Reply:
I don't follow. If you supply a URL, then the file open will fail with `FileNotFoundError`, which will then get caught by the except clause, which will try to open it using `requests`

Reply:
![image](https://user-images.githubusercontent.com/63688713/90286774-85abe980-de44-11ea-929d-30c83810491b.png)

Reply:
Just tested without catching the `OSError`:

![Screen Shot 2020-08-14 at 12 44 05 PM](https://user-images.githubusercontent.com/51207431/90287101-07434d80-de2c-11ea-834e-21fe1e56bd69.png)

Reply:
It looks like the behaviour differs on Windows. Alright, easy fix.

Reply:
Resolved in the newest commit